compiler/rustc_target/src/spec/crt_objects.rs

```diff
@@ -74,7 +74,7 @@ pub(super) fn pre_musl_fallback() -> CrtObjects {
 }
 
 pub(super) fn post_musl_fallback() -> CrtObjects {
-    all("crtn.o")
+    all(&["crtn.o", "rsend.o"])
```

Comment:
Feel free to r=me if you think this concern is unwarranted.

Reply:
I think you're right, it's safer to link the normal crtbegin/crtend objects, rather than reimplementing them. I've adjusted things to copy the object files and link them according to the table at the top of the file.

Reply:
This is exactly what I was going to suggest.

(The comment higher in this file says that we are not linking crtbegin/crtend, but it can probably be kept as is because we are still not linking them on other targets supporting self-contained mode.)